<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>253</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>-1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openPathButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Прикрепить файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createGoldenRecordButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Сформировать золотую запись</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathLine">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>361</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
